api/v1beta1/common_types.go

```diff
@@ -169,9 +169,10 @@ type ServiceOptions struct {
 	// +optional
 	SessionAffinity string `json:"sessionAffinity,omitempty"`
 
-	// sessionAffinityConfig contains the configurations of session affinity.
 	// +optional
-	SessionAffinityConfig corev1.SessionAffinityConfig `json:"sessionAffinityConfig,omitempty"`
+	// +kubebuilder:default=None
```

Comment:
These (the default and enum lines) were meant to be put up above with `sessionAffinity`, not `sessionAffinityConfig`.